opencog/cython/opencog/atomspace.pxd

```diff
@@ -82,7 +82,7 @@ cdef extern from "opencog/atomspace/atom_types.h" namespace "opencog":
     cdef Type NOTYPE
 
 # Handle
-ctypedef long UUID
+ctypedef public long UUID
```

Comment:
Okay, I submitted pull request #1372 which has to perform some very ugly incantations to get the reference to the AtomPtr back to the C++ Handle constructor.